### ISSUE TITLE
PyFX does not work with overrides==6.1.0

### ISSUE THREAD

**Issue by BarclayII** [cielong/pyfx#56]
**Describe the bug**
Running PyFX with the latest `overrides` version (which is 6.1.0 currently) throws an error:
```
TypeError: `ArrayNode.prev_child: key overriding must be a supertype of `<class 'inspect._empty'>` but is `<class 'str'>`
```
The reason is that `ArrayNode`'s `prev_child` method overrides the parent class' `prev_child` method, which have a different type hint, and `overrides` now check the type hints starting from some version later than 4.1.2.

**To Reproduce**
1. Install PyFX from pip from a clean environment, which by default installs the latest `overrides` package as dependency.
2. Open any json file.

**Expected behavior**
It should not throw an error.

**Additional context**
Downgrading `overrides` to 4.1.2 solves the problem.  Did not test other versions.

**Comment by cielong** [comment]
Thanks for digging into the issue!
I thought I have fixed a similar issue in commit [4913ecd7bc6abc3ee05d1ff9f41c656bfb449fe1](https://github.com/cielong/pyfx/commit/4913ecd7bc6abc3ee05d1ff9f41c656bfb449fe1).

Maybe I overlooked one and did not fix it then.

May I know which version you're using? I'll take a look to fix it

**Comment by cielong** [comment]
Just released 0.1.0-beta.2, in which I fixed the dependency version in the latest version. When you got a chance, can you help me test it?

Thanks again for using the tool!

**Comment by BarclayII** [comment]
I was using the latest release from PyPI.  I think your mentioned commit fixed it.

Thanks!
